tests/shell/shell_recorder/script.dat

```diff
@@ -0,0 +1,13 @@
+Storage: dump
+Mountpoint: system/testmount/test
+File: /tmp/test.dump
+MountArgs:
```

Comment:
Why do you think untrusted code could be a problem? Isn't everything within the repo?